### DISCUSSION TITLE
[Feature Request] Font Customization for Code Editor

### DISCUSSION THREAD

**Discussion by ghost** [standardnotes/forum#2511] [Suggestions] [answered]
Currently the code editor lacks font customization. I find it a bit hard to read some of the code I am adding, mostly if the code is very concentrated, the current font is too "squarish" if that's the write adjective and makes it hard to read it. It would make absolute sense to allow the user to customize the font and change it.

**Comment by ghost** [comment]
@mobitar I fixed this readability problem by putting line-height: 1; in the css of the extension folder ! It would help the user experience of everyone if you did that to be honest.

**Comment by atmoio** [comment]
>  in the css of the extension folder 

What do you mean by that? Are you running a self hosted theme?

Also, can you share before and after screenshots? Testing locally I don't see that it's too condensed, so I wonder if it's a system-specific thing..

**Comment by ghost** [comment]
@mobitar The issue is that if you have a script in the code editor that is too concentrated, the css doesn't give each line a sufficient distance and it's really hard to read it. What I did to fix this was, I went to my local "code editor" extension folder in the dist and edited the css file and under ".CodeMirror {" I placed   "line-height: 1;" now it is as readable as it should be !

**Comment by atmoio** [comment]
Ah I see you placed it in the offline editor. That will work. But not for web, if that matters.

**Comment by myreli** [comment, marked as answer]
The font is customizable trough the `--sn-stylekit-monospace-font` CSS variable using a [custom theme](https://standardnotes.com/help/plugins/themes).
